webcompat/static/js/lib/wizard/steps/submit.js

```diff
@@ -8,31 +8,16 @@
 
 import $ from "jquery";
 import { showContainer } from "../ui-utils.js";
-import { isGithubUserNameValid } from "../validation.js";
```

Comment:
Do we want to delete `isGithubUserNameValid` as well? I don't think it's used anywhere else.

Reply:
Good point, missed that!

Reply:
ready for another look @miketaylr :)